fdm-core/src/cultivation.d.ts

```diff
@@ -30,5 +30,21 @@ export interface cultivationPlanType {
             p_app_date: schema.fertilizerApplicationTypeSelect["p_app_date"]
             p_app_id: schema.fertilizerApplicationTypeSelect["p_app_id"]
         }>
+        harvests: Array<{
+            b_id_harvesting: schema.cultivationHarvestingTypeSelect["b_id_"]
+            b_harvesting_date: schema.cultivationHarvestingTypeSelect["b_harvesting_date"]
+            harvestables: Array<{
+                b_id_harvestable: schema.harvestableAnalysesTypeSelect["b_id_harvestable"]
+                harvestable_analyses: Array<{
+                    b_lu_yield: schema.harvestableAnalysesTypeSelect["b_lu_yield"]
+                    b_lu_n_harvestable: schema.harvestableAnalysesTypeSelect["b_lu_n_harvestable"]
```

Comment:
do you also need crop harvest index beign ratio harvestable/residue?

Reply:
That should be a cultivation property that the user can modify